react-ui/src/App.js

```diff
@@ -44,6 +45,10 @@ class App extends Component {
           {/* <Redirect from="/" to="dashboard/" /> */}
         </Route>
 
+        <Route path="/workerinputavailability" component={WorkerInputpage}> 
```

Comment:
Hi Kyle,
I would remove this and add it to the `Dashboard` component rather than the App section since it should only be available after the user has logged into the system.

Reply:
done